apps/testrunner/runner.py

```diff
@@ -5,22 +5,47 @@
 import sys
 import six
 
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
 from .logger import logger
 from .test_suites import SimpleTestSuite
 
 
+class AutoDiscoverFailed(ImproperlyConfigured):
+    pass
+
+
 class TaskRepo(object):
     """
     Encapsulates HTTP tasks API
     """
-    # todo: get rid of hardcoded URL here
-    URL = 'http://localhost:8000/api/v1/tasks'
+
+    API_SERVER = None
+    TASKS_API_URL = None
+    RESULTS_API_URL = None
+
+    def __init__(self):
+        # Auto Discover API URIs
+        self.API_SERVER = settings.SPARROW_TEST_RUNNER['api_server']
+        api_descr = {}
+
+        try:
+            req = requests.get(self.API_SERVER)
+            req.raise_for_status()
+
+            api_descr = req.json()
+
+            self.TASKS_API_URL = api_descr['tasks']
+            self.RESULTS_API_URL = api_descr['results']
+        except Exception as ex:
+            raise AutoDiscoverFailed('Cannot find API URIs: {0}'.format(api_descr)) from ex
 
     def acquire(self):
-        url = '{}/fetch/'.format(self.URL)
+        url = '{}fetch'.format(self.TASKS_API_URL)
```

Comment:
Right. It works but trailing slash should be there: bd4647a3dc1a53b889298007fb5eeb10a0935bef